lib/ravelin/errors/api_error.rb

```diff
@@ -0,0 +1,19 @@
+module Ravelin
+  class ApiError < StandardError
+    attr_reader :response, :status, :error, :error_message, :validation_errors
+
+    def initialize(response)
+      @response           = response
+      @status             = response.status
+      @error              = response.body.fetch('Error', nil)
+      @error_message      = response.body.fetch('message', nil)
+      @validation_errors  = response.body.fetch('validationErrors', [])
+    end
+
+    def to_s
```

Comment:
Probably not worth changing at this stage, but for future reference you may want to define the `message` method in a case like this:

```ruby
class ApiError < RuntimeError
  def message
    "Some extra info"
  end
end

raise ApiError
# api_error.rb:7:in `<main>': Some extra info (ApiError)
```